src/components/sbb-card/sbb-card.scss

```
@@ -0,0 +1,130 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-xxs);
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-3x);
+  --sbb-card-padding-block-end: var(--sbb-card-padding-block-start);
+
+  position: relative;
+  overflow: hidden;
+  border-radius: var(--sbb-border-radius-4x);
+  transition: box-shadow var(--sbb-animation-duration-6x) var(--sbb-animation-easing);
+  z-index: 0;
+
+  &::before,
+  &::after {
+    position: absolute;
+    content: '';
+    inset: 0;
+  }
+
+  &::before {
+    background-color: var(--sbb-color-white-default);
+    z-index: -3;
+  }
+
+  &::after {
+    background-color: transparent;
+    z-index: -1;
+  }
+
+  @include ifForcedColors {
+    outline: var(--sbb-border-width-1x) solid CanvasText;
+  }
+}
+
+::slotted(sbb-card-badge) {
+  z-index: -2;
+}
+
+:host([size='s']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host([size='m']) {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-s);
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host([size='l']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-4x);
+}
+
+:host([size='xl']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxs);
+}
+
+:host([size='xxl']) {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-s);
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-s);
+}
+
+:host(.sbb-card--has-badge[size='m']),
+:host(.sbb-card--has-badge[size='l']),
+:host(.sbb-card--has-badge[size='xl']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-6x);
+}
+
+:host(.sbb-card--has-badge[size='m']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host(.sbb-card--has-badge[size='l']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-fixed-4x);
+}
+
+:host(.sbb-card--has-badge[size='xl']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-responsive-xxs);
+}
+
+:host(.sbb-card--active) {
+  &::after {
+    border: var(--sbb-border-width-2x) solid var(--sbb-color-smoke-default);
+    border-radius: var(--sbb-border-radius-4x);
+  }
+}
+
+.sbb-card__content {
+  display: block;
+  transition: transform var(--sbb-animation-duration-6x) var(--sbb-animation-easing);
+}
+
+.sbb-card__link {
+  display: block;
+  color: var(--sbb-color-black-default);
```

Would inherit be an option so we don't have to specify a specific color?
```suggestion
  color: inherit;
```